lib/ngx-wasm-rs/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "ngx-wasm"
+name = "ngx-wasm-rs"
```

Comment:
The Rust API guidelines recommend against `-rs` for the actual crate names:

https://rust-lang.github.io/api-guidelines/naming.html

> Crate names should not use -rs or -rust as a suffix or prefix. Every crate is Rust! It serves no purpose to remind users of this constantly.

So you see many projects in the wild where the repo is `foo-rs` but the crate is `foo`.

Reply:
Given we will not publish this create to crates.io I preferred going for consistency so it would help distinguish it from the different Rust crates we have: "`ngx-wasm-rs` is the .so lib we extra features we embed into our builds". I looked at `ngx-wasm` and its crate name is `ngx`; I'll revert this for now but a rename of both of these crates (or even a merge) is incoming in any case because all these names are more confusing than anything (also we should align with the Lua lib and name it with `wasmx` instead of `wasm`).